src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1824,6 +1824,15 @@ properties:
                 type: string
                 fits_keyword: R_DRZPAR
                 blend_table: True
+          emi:
+            title: MIRI Reference Waves Table for EMI correction
+            type: object
+            properties:
+              name:
+                title: MIRI Reference Waves Table for EMI correction
```

Comment:
I know it's been said that foolish consistency is the hogoblin of little minds, but I'm OK with having a little mind, and hence suggest we change this a bit to maintain consistency with other R_xxxxxx keyword titles, so how about "MIRI EMI correction reference file name". That also happens to match what's been proposed in the Keyword Dictionary ticket for adding this keyword to the official JWST dictionary (https://jira.stsci.edu/browse/JWSTKD-549).

Reply:
done :)